requirements.txt

```diff
@@ -22,3 +22,5 @@ gitdb==4.0.9
 Werkzeug==1.0.1
 WTForms==2.3.3
 MarkupSafe<=2.0.1
+jinja2==3.0.2
+itsdangerous==2.0.1
```

Comment:
I would like to understand why these two new dependencies are explicitly mentioned in requirements.txt?

Reply:
Upon installing the dependencies through the current requirements.txt, it currently installs jinja2==3.1.2 and since we are using Flask 1, we need jinja2<3.1.0 to import `escape` from it, or we can import `escape` through `markupsafe`. The error snapshot is posted along with PR description, and solve using [this](https://stackoverflow.com/questions/71718167/importerror-cannot-import-name-escape-from-jinja2) link.

Reply:
A similar issue was faced due to updated dependency `itsdangerous` and solved using [this link](https://stackoverflow.com/questions/71189819/python-docker-importerror-cannot-import-name-json-from-itsdangerous).